web-app/src/main/java/com/infoshare/webapp/controller/UploadController.java

```diff
@@ -0,0 +1,47 @@
+package com.infoshare.webapp.controller;
+
+import com.infoshare.webapp.model.Questions;
+import com.infoshare.webapp.service.FileJSONService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+@Controller
+public class UploadController {
+
+    @Autowired
+    FileJSONService fileJSONService;
```

Comment:
```suggestion
    @Autowired
    private final FileJSONService fileJSONService;
```
zalecaną metodą wstrzykiwania zależności jest wstrzykiwanie przez konstruktor

Reply:
poprawione